### PR TITLE
Double Elimination Calculator

### PR DESCRIPTION
**Pull request by tommyvernieri**
Calculates TGP for double elimination head to head brackets. Uses a corrected formula that comes in a bit higher than the formula used by the IFPA from WPPR v5.3 to v6.0. Pending IFPA approval.

### CODE REVIEW

**Comment by tommyvernieri** [comment]
@gameboy9 Please take a look and let me know what you think.
There's more info in this spreadsheet: https://docs.google.com/spreadsheets/d/1uoRVCvH4xGuh1MMtIQKRkmJNGHxcF-wfoGBrDuDLwXY/edit?usp=sharing

File: doubleelim.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+			<option value="2">Best-of-3</option>
+			<option value="3">Best-of-5</option>
+			<option value="4">Best-of-7</option>
+		</select>
+	</div>
+	
+	<div class="w3-row w3-container" style="margin-top: 16px;">
+		<div class="w3-col s12 m12 l2"><b>Winner's bracket match expected games</b></div>
+		<div class="w3-col s12 m12 l4">
+			<input type="number" id="evw" name="evw" min="1" max="50" value="2.5" />
```
Comment (gameboy9) [inline doubleelim.html:41]: You can set up a step by 0.5; let's do that, considering the default here.  (it's currently at its default, which is 1.0)

(repeat this for the Loser's bracket)

**Review by gameboy9 — CHANGES_REQUESTED** [review]
Just a couple of things I found.  I will then approve and merge when the IFPA is OK with your changes to the calculation, since it's different from WPPR 5.x.

Thanks for doing this!

File: doubleelim.html (hunk 1 above)
Comment (tommyvernieri) [reply doubleelim.html]: Done. I also normalized spacing in both files and I added a "None - Single Elimination" option for the loser's bracket length.

File: doubleelim.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    <meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>TGP Knockout Calculator</title>
+</head>
+
+<body>
+	<div w3-include-html="menu.html"></div>
+	
+	<div class="w3-container w3-row" style="margin-top: 64px;">
+		<input type="number" id="playerCount" name="playerCount" min="3" max="1000" value="8" />
```
[the remark this answers is not in the document]
Comment (tommyvernieri) [reply doubleelim.html]: Done

**Comment by tommyvernieri** [comment]
Approved by the IFPA. They have updated their TGP guide. @gameboy9 please merge whenever you're ready.

**Comment by gameboy9** [comment]
It's done; thanks for doing this, Tommy!

**Pull request merged**